index.js

```diff
@@ -89,7 +89,7 @@ function parse(html, doc) {
   }
 
   // wrap map
-  var wrap = map[tag] || map._default;
+  var wrap = map.hasOwnProperty(tag) ? map[tag] : map._default;
```

Comment:
Should this be `Object.prototype.hasOwnProperty.call(map, tag)` for the same reason?

```suggestion
  var wrap = Object.prototype.hasOwnProperty.call(map, tag) ? map[tag] : map._default;
```

Reply:
Thanks a lot for your reply and feedback @stephenmathieson !
It's great that this library is still maintained :)

Yes, sure, this sounds good to me! I honestly didn't know there was a difference until now! Thanks again!
TIL

Reply:
Looking at this again, I realize that we own `map`, so this change was unnecessary 🤦